controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -114,59 +115,60 @@ def test_create_volume_with_empty_space_efficiency_success(self):
         self._test_create_volume_with_space_efficiency_success("")
 
     def test_create_volume_raise_illegal_name_for_object(self):
-        self.mediator.client.cmd.vol_create.side_effect = [xcli_errors.IllegalNameForObjectError("", "vol", "")]
+        self.mediator.client.cmd.vol_create.side_effect = [xcli_errors.IllegalNameForObjectError("", "volume", "")]
         with self.assertRaises(array_errors.IllegalObjectName):
-            self.mediator.create_volume("vol", 10, None, "pool1")
+            self.mediator.create_volume("volume", 10, None, "pool1")
 
     def test_create_volume_raise_volume_exists_error(self):
-        self.mediator.client.cmd.vol_create.side_effect = [xcli_errors.VolumeExistsError("", "vol", "")]
+        self.mediator.client.cmd.vol_create.side_effect = [xcli_errors.VolumeExistsError("", "volume", "")]
         with self.assertRaises(array_errors.VolumeAlreadyExists):
-            self.mediator.create_volume("vol", 10, None, "pool1")
+            self.mediator.create_volume("volume", 10, None, "pool1")
 
     def test_create_volume_raise_pool_does_not_exists_error(self):
         self.mediator.client.cmd.vol_create.side_effect = [xcli_errors.PoolDoesNotExistError("", "pool", "")]
         with self.assertRaises(array_errors.PoolDoesNotExist):
-            self.mediator.create_volume("vol", 10, None, "pool1")
+            self.mediator.create_volume("volume", 10, None, "pool1")
 
     def test_create_volume_raise_no_space_error(self):
         self.mediator.client.cmd.vol_create.side_effect = [
             xcli_errors.CommandFailedRuntimeError("", "No space to allocate to the volume", "")]
         with self.assertRaises(array_errors.NotEnoughSpaceInPool):
-            self.mediator.create_volume("vol", 10, None, "pool1")
+            self.mediator.create_volume("volume", 10, None, "pool1")
 
     @patch.object(XIVArrayMediator, "_generate_volume_response")
     def test_create_volume__generate_volume_response_raise_exception(self, response):
         response.side_effect = Exception("err")
         with self.assertRaises(Exception):
-            self.mediator.create_volume("vol", 10, None, "pool1")
+            self.mediator.create_volume("volume", 10, None, "pool1")
 
     def test_copy_to_existing_volume_from_snapshot_succeeds_with_resize(self):
-        vol_name = "vol"
-        src_snap_name = "snap"
-        src_snap_capacity_in_bytes = 500
-        min_vol_size_in_bytes = 1000
+        volume_name = "volume"
+        src_snapshot_name = "snapshot"
+        src_snapshot_capacity_in_bytes = 500
```

Comment:
```suggestion
        source_snapshot_name = "snapshot"
        source_snapshot_capacity_in_bytes = 500
```

in all relevant places

Reply:
I agree, but I think this PR is already large enough as it is...

Reply:
CSI-2896